src/_pytest/assertion/util.py

```diff
@@ -112,6 +113,18 @@ def isattrs(obj):
     return getattr(obj, "__attrs_attrs__", None) is not None
 
 
+if attrs_has_eq:
+
+    def attrsfieldhaseq(a):
+        return a.eq
+
+
+else:
+
+    def attrsfieldhaseq(a):
+        return a.cmp
```

Comment:
black really loves its whitespace doesn't it :man_facepalming:

Reply:
Black and white, ying and yang!